docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Authentication & authorization"
+menuPosition: 3
+editor: tylerbutler
+---
+
+Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
+important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are
+primarily a concern of the *driver* layer (see [Architecture]({{< relref "architecture.md" >}})). This means that the
+specifics of authentication could differ based on the Fluid service and its driver.
+
+The information below is based on Azure Fluid Relay service (FRS) but also applies to
+[Tinylicious]({{< relref "tinylicious.md" >}}). Other Fluid services may differ.
```

Comment:
It doesn't apply to tinylicious. tinylicious is inherently insecure by design and there's no tenant secret. Probably worth having two sections? 

```suggestion
```